README.md

```diff
@@ -35,7 +35,7 @@ providing the following features:
 
 The following features are high priority and planned for the very near future:
 
-* Ability to distribute builds to multiple build slaves
+* Ability to distribute builds to multiple build replicas
```

Comment:
`replica` is not a suitable name, because nothing is actually being replicated. Just "multiple build machines" would be sufficient I think to avoid the slave-terminology.

It also would match the terminology on Nix, i.e., `nix.buildMachines` (from [here](https://nixos.wiki/wiki/Distributed_build)).

Reply:
good call, I wasn't sure if `replica` was the correct word.